### ISSUE TITLE
Consider adding docker suport

### ISSUE THREAD

**Issue by EugenMayer** [livelyapps/pluploader#32]
Then in your readme all you need to do is

```bash
docker run -v "$(pwd)":/workdir eugenmayer/pluploader list
```
In the folder mounted there most be the .pluprc or whereever, could also be 

```
docker run -v "$HOME/.pluprc":/workdir eugenmayer/pluploader list
```

No need for an installation for your users

For you: Just build the image using 

```Dockerfile
FROM python

RUN pip3 install pluploader \
 && apt-get update \
 && apt-get install -y vim \
 && mkdir -p /workdir

WORKDIR /workdir

ENTRYPOINT [ "pluploader" ]
```

(use your namespace)
```
docker build . -t eugenmayer/pluploader
```

and push it on dockerhub. I pushed mine already, so you can already try the above

**Comment by craftamap** [comment]
Great suggestion! I will implement this soon after I'm back in the office!

**Comment by EugenMayer** [comment]
great you like it.

created #33 for the dockerfile, you will need to add the readme part yourself since i'am not aware of your dockerhub namespace

**Comment by craftamap** [comment]
While #33 is merged now, there are still todos (for me) regarding docker support:
 - [ ] Adding docker image to the readme
 - [ ] Push pluploader v0.4.1 to Dockerhub
 - [ ] Add pipeline pushing the building and publishing docker image on pypi release

**Comment by craftamap** [comment]
We should also think about publishing the docker image here, since github allows to publish docker images, too.

**Comment by EugenMayer** [comment]
if you do not have an public ci server you want to use for this and want to keep this thing out of your companies ci, use the docker.hub builder which is very easy to configure as a trigger on tags or changes (you will want to have tags to integrate with pypi i guess)

I'am not sure about githubs docker registry, esp. the anon access part, but you should publish on hub as a backup to ensure existing proxy setups work without issues, IMHO

**Comment by craftamap** [comment]
@EugenMayer I use the github ci already for the pluploader, so no problem here. But thanks for the help!

**Comment by craftamap** [comment]
This ticket will be closed now as #35 was merged now!

The README needs to be updated still, but you can already find the pluploader here: https://hub.docker.com/repository/docker/craftamap/pluploader/
